<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundSpeedOfSeaWaterClass</class>
 <widget class="QMainWindow" name="SoundSpeedOfSeaWaterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>声速经验公式计算器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblFunType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>经验公式类型：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxFunType">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblTemp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>温度/°C：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSalt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>盐度/ppt：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDepth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>深度/m：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTemp">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSalt">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDepth">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblSpeed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>理论声速/(m/s)：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSpeed">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCalculate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>计算</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SoundSpeedOfSeaWater.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbxFunType</sender>
   <signal>activated(int)</signal>
   <receiver>SoundSpeedOfSeaWaterClass</receiver>
   <slot>ComboBox_Actived(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCalculate</sender>
   <signal>clicked()</signal>
   <receiver>SoundSpeedOfSeaWaterClass</receiver>
   <slot>btnCalculate_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnReset</sender>
   <signal>clicked()</signal>
   <receiver>SoundSpeedOfSeaWaterClass</receiver>
   <slot>btnReset_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ComboBox_Actived(int)</slot>
  <slot>btnCalculate_Clicked()</slot>
  <slot>btnReset_Clicked()</slot>
 </slots>
</ui>
